HelpDesk.App.Global/appsettings.json

```diff
@@ -14,7 +14,7 @@
   },
   "AllowedHosts": "*",
   "RsaKeysPath": {
-    "Public": "D:\\BOAuthRsaKeys\\HdProduction\\public.json",
-    "Private": "D:\\BOAuthRsaKeys\\HdProduction\\private.json"
+    "Public": "/Users/aboho/Desktop/hd-production/dashboard-api/HdProduction.Dashboard.Api/RSAKeys/public.json",
```

Comment:
you are pidor
never do like this(
create your own env or just not commit that

Reply:
Sorry 😅
It shouldn't have gotten into git

Reply:
ok, you've added your env.. but what about changes in this file??